draft-wing-settle-public-key-hash.md

```diff
@@ -233,7 +233,20 @@ connection is made to that address, those are also considered
 
 # Security Considerations
 
-TODO: write more on security considerations
+Due to operational challenges in key rotation, some servers may need to
+maintain static public/private key pairs over long periods. This introduces a
+tradeoff: while static keys expose servers to risks of private key compromise
+and active attacks (e.g., traffic interception or server impersonation), they
+still provide better security than unencrypted communication. The current approach
```

Comment:
The theft of the private key that enables active attacks, but that does not mean key rotation prevents active attacks.

```suggestion
, encrypted communication provides better security than unencrypted communication. The current approach
```